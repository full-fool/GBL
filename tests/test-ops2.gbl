class mymain extends Main{
int main()
{
  printlnb(true);
  printlnb(false);
  printlnb(true and true);
  printlnb(true and false);
  printlnb(false and true);
  printlnb(false and false);
  printlnb(true or true);
  printlnb(true or false);
  printlnb(false or true);
  printlnb(false or false);
  printlnb(not false);
  printlnb(not true);
  printlni(-10);
  printlni(-42);
}
}