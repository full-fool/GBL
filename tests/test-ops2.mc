int main()
{
  println(true);
  println(false);
  println(true and true);
  println(true and false);
  println(false and true);
  println(false and false);
  println(true or true);
  println(true or false);
  println(false or true);
  println(false or false);
  println(not false);
  println(not true);
  println(-10);
  println(-42);
}
