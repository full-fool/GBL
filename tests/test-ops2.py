import sys
class mymain:
    def __init__(self):
        pass

    def main(self):
        print (True)
        print (False)
        print ((True) and (True))
        print ((True) and (False))
        print ((False) and (True))
        print ((False) and (False))
        print ((True) or (True))
        print ((True) or (False))
        print ((False) or (True))
        print ((False) or (False))
        print (not (False))
        print (not (True))
        print ( - (10))
        print ( - (42))
        pass


if __name__ == "__main__":
    _main = mymain()
    _main.main()

