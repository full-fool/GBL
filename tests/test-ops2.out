True
False
True
False
False
False
True
True
True
False
True
False
-10
-42
